frontend/src/components/Contexts/AuthProvider.tsx

```diff
@@ -28,7 +28,5 @@ export const AuthProvider = (props: Props) => {
         return () => clearInterval(id)
     }, [])
 
-    VerifyToken()
```

Comment:
Why is this removed?

Reply:
It creates an error, shouldn't call callback functions in component definition